engine/src/main/java/org/destinationsol/common/SolMath.java

```diff
@@ -15,49 +15,72 @@
  */
 package org.destinationsol.common;
 
-import java.util.ArrayList;
-
-import org.destinationsol.Const;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.badlogic.gdx.math.MathUtils;
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.utils.JsonValue;
 import com.badlogic.gdx.utils.Pool;
+import org.destinationsol.Const;
+
+import java.util.ArrayList;
 
 /**
- * A class with helpful mathematical functions
+ * This class presents a collection of handy mathematical as well as programmatical functions, for working with angles,
+ * distances, speeds, vectors.
  */
 public class SolMath {
-    private static Logger logger = LoggerFactory.getLogger(SolMath.class);
+    /**
+     * The pi number constant.
+     */
     public static final float PI = MathUtils.PI;
+    /**
+     * Multiply radians by this number to convert them to degrees.
+     */
     public static float radDeg = MathUtils.radDeg;
```

Comment:
`Multiply radians by this number to convert them to degrees.` -> `Conversion factor to transform radians to degrees.`

Reply:
Aren't javadoc for explaining what it is and how to use it? It might be just me, but given your version, I'd still have to look at the actual contents of the var to understand what it is and how to use it. Also, since the variable is basically just copy-pasted from libGDX, so is the javadoc of mine.